tests/importable_test.py

```diff
@@ -136,6 +136,29 @@ def __str__(self) -> str:
     ),
 )
 
+supported_inventory_plugins = (
```

Comment:
Merge them. From the runtime perspective, they are the same and there shouldn't be any difference in this.
```suggestion
```